spring-integration-core/src/test/java/org/springframework/integration/transformer/MethodInvokingTransformerTests.java

```diff
@@ -37,10 +38,10 @@
  * @author Mark Fisher
  * @author Artem Bilan
  */
-public class MethodInvokingTransformerTests {
+class MethodInvokingTransformerTests {
 
 	@Test
-	public void simplePayloadConfiguredWithMethodReference() throws Exception {
+	void simplePayloadConfiguredWithMethodReference() throws Exception {
```

Comment:
Why change from `public`?

Reply:
Because it is now JUnit 5 test and that one doesn’t require `public` on methods. My point is that eventually we have to come fully only to JUnit 6 or already 6. The 4th one might just go EOL some day...

Reply:
Just because public is not required, I don't see why it's worth the effort to change it all over the place; seems like unnecessary work to me.